draft-ietf-quic-transport.md

```diff
@@ -2698,24 +2698,24 @@ frames are explained in more detail in {{frame-formats}}.
 | Type Value  | Frame Type Name      | Definition                     |
 |:------------|:---------------------|:-------------------------------|
 | 0x00        | PADDING              | {{frame-padding}}              |
```

Comment:
Considering that CONNECTION_CLOSE has a frame type field, where 0x0 means that the error is not frame related, wouldn't it make sense to not use 0x0 at all for frames.

Reply:
0 was used for padding intentionally.  calloc() and write-in-place implementations are easier this way.